### ISSUE TITLE
Lines in TOC are cut off

### ISSUE THREAD

**Issue by hahv** [ahrm/sioyek#161]
Thank you for this cool software. 
**One minor issue:**
- Is that possible to show all text (in a line) in the table of content instead showing ellipsis (...) to cut off the text (img below).
![Capture](https://user-images.githubusercontent.com/8085290/156704445-21e00d49-2269-4086-bccc-4a25d5543074.PNG)

**Comment by ahrm** [comment]
I am not able to reproduce your issue on my system. Which operating system and which build of sioyek are you using?

**Comment by hahv** [comment]
Sorry, let me add the details for the issues:
+ OS: Win 10 Pro 64-bit (Build 19041)
+ Sioyek: v.1.1 (lastest)
+ Tested File (with TOC): 
[Feichtenhofer_SlowFast_Networks_for_Video_Recognition_ICCV_2019_paper.pdf](https://github.com/ahrm/sioyek/files/8194317/Feichtenhofer_SlowFast_Networks_for_Video_Recognition_ICCV_2019_paper.pdf)

The TOC are normal when open with Foxit Reader as image below:
![Capture1](https://user-images.githubusercontent.com/8085290/156958050-0962e9ff-c59f-4340-84a7-277edce380a0.PNG)

Thank you!

**Comment by ahrm** [comment]
I was still not able to reproduce this issue. Have you changed any particular setting in your `prefs.config`?

**Comment by hahv** [comment]
You are right. I did change some configs, but I think it is not likely to relate to this issues. Anyway all the changes are following:
+ in `prefs.config` files (add at the end of the file,i.e after default configs):

```
# if set, we use this font for UI text
ui_font Source Code Pro
font_size 15
```
+ in `prefs_user.config` files
```
status_bar_font_size    14
rerender_overview	1
page_separator_width     1
page_separator_color	0.9 0.9 0.9
custom_background_color              0.984 0.941 0.850
custom_text_color                    0.370 0.294 0.196
startup_commands             toggle_custom_color
```
+ in keys_user.config

`toggle_custom_color     <f3>
`
Maybe font size changes cause that problem?

**Comment by ahrm** [comment]
Should be fixed in 4c9ad227f4d896dbc3baeb8d2395947670314a85.
